test/server/cards/01_SOR/events/Vigilance.spec.ts

```diff
@@ -100,7 +108,7 @@ describe('Vigilance', function() {
                 });
             });
 
-            it('discards 6 cards from an opponent\'s deck dealing 6 damage to his base and heal 0 damage from base.',
+            it('discards 6 cards from an opponent\'s deck dealing 0 damage to his base and heal 0 damage from base.',
```

Comment:
nit: could we just remove the reference to damage completely? Imo having it say 0 makes it sound like there would normally be an expectation that damage happens, which isn't true